packages/ladle/lib/app/exports.ts

```diff
@@ -27,7 +31,14 @@ type ReactNodeWithoutObject =
   | null
   | undefined;
 
-export { useLadleContext, ActionType, ThemeState, ModeState };
+export {
+  useLadleContext,
+  ActionType,
+  ThemeState,
+  ModeState,
+  getMetaJsonString,
+  getMetaJson,
```

Comment:
The user needs to provide `entryData` 

```js
getMetaJson(entryData)
```

but there is no way to get those from outside. We should export fn that does that internally (calls https://github.com/tajo/ladle/blob/main/packages/ladle/lib/cli/vite-plugin/parse/get-entry-data.js ...).